app/src/main/res/drawable/primary_button_green.xml

```diff
@@ -2,19 +2,19 @@
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:state_pressed="true">
         <shape android:shape="rectangle">
-            <solid android:color="@color/test_app_green_dark" />
+            <solid android:color="#72acaa" />
```

Comment:
Can we use the defined colors as well? (as before)

Reply:
@tschuls done and tested with low api version 👍